src/coreclr/nativeaot/Test.CoreLib/src/System/RuntimeTypeHandle.cs

```diff
@@ -10,17 +10,17 @@ namespace System
     [StructLayout(LayoutKind.Sequential)]
     public struct RuntimeTypeHandle
     {
-        private EETypePtr _pEEType;
+        private IntPtr _pEEType;
```

Comment:
What motivated this change? We have another copy at `src\coreclr\nativeaot\Runtime.Base\src\System\RuntimeTypeHandle.cs` that can be updated as well.

Also, while you are on it - could you please delete this comment https://github.com/dotnet/runtime/blob/4860b6c28f4c39b64b600f16b428931bf3dc4455/src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeTypeHandle.cs#L19-L23 and move `private IntPtr _value;` to the top in that file?

Reply:
> What motivated this change? 

I was looking at what stack traces do allocations and at some point I had doubts that stacks are correct. So I tried to look at `optimized=false` traces. It was mostly the same, in terms of allocations, except a few cases of boxing, that otherwise are optimized. 

What surprised me is the wrapping/unwrapping of the RuntimeTypeHandle - it was near the top. It looks like a common operation, so I "fixed" it to not clutter the profile.  
This change is not required, since JIT ca optimize this away, but I figured it is better if I keep "fixed" variant.